processor/k8sattributesprocessor/internal/kube/client.go

```diff
@@ -210,6 +210,7 @@ func New(
 func (c *WatchClient) Start() error {
 	synced := make([]cache.InformerSynced, 0)
 
+	waitForInformers := []cache.SharedInformer{}
```

Comment:
Do we need a second array for the informers? I would try to just use the `synced` directly excluding the Pod's sync status.